README.md

```diff
@@ -174,6 +174,7 @@ configuration options.
     * processes
     * kernel (/proc/stat)
     * kernel (/proc/vmstat)
+    * linux_sysctl_fs (/proc/sys/fs)
```

Comment:
As I was adding this line, I noticed the naming isn't in line with the other linux-only metrics. The other 2 linux-only measurements are named "kernel". We could call this "kernel_sysctl_fs". I don't much like the name, as "kernel" does not indicate that it's linux-specific. But perhaps this is just something we fix in [2.0](https://github.com/influxdata/telegraf/issues/2320)